### DISCUSSION TITLE
Different results with markitdown?

### DISCUSSION THREAD

**Discussion by MJonker** [Azure-Samples/azure-openai-entity-extraction#3] [General]
https://github.com/microsoft/markitdown

I was curious if you also tried markitdown instead of mymupdf?

**Comment by pamelafox** [comment]
I haven't tried it, no! I could try it for a handful of PDFs, but I imagine that the quality may depend on the particular format of the PDF. If you do try it, feel free to send a PR to the repo with an example.
